src/dmd/link.d

```diff
@@ -160,6 +160,30 @@ public int runLINK()
     const phobosLibname = global.params.betterC ? null :
         global.params.symdebug ? global.params.debuglibname : global.params.defaultlibname;
 
+    if (phobosLibname)
```

Comment:
This block may not even be worth trying to support.  We could just rely on the config to do this as well.